contracts/strategies/MobiusStrategy.sol

```diff
@@ -41,12 +41,15 @@ contract MobiusStrategy is Ownable, IStrategy {
     /// @notice celo token
     IERC20 public immutable celo;
 
+    /// @notice pool address
+    IMobiPool public immutable pool;
+
+    /// @notice token index in the pool.
+    uint8 public immutable inboundTokenIndex;
```

Comment:
nothing wrong here, but maybe we just want to keep variable types grouped together and move this one after `lpToken`.
And about the line  `IERC20 public lpToken;`:

can we make it `immutable` as well?

Reply:
no because it is initialized in an if hence